### DISCUSSION TITLE
Surprised my slingbox changed it's port?

### DISCUSSION THREAD

**Discussion by mdehrlich** [GerryDazoo/Slinger#658] [General]
After running slinger for over a year(s), I was surprised tonight that I could not connect on slingplayer.  After checking my router, I found the port number had changed from 5201 to 5203!  I changed the config file, and it all works fine again.  Any idea why it changed?  If it happens while I am traveling, how can I find out the value of the new port number?  Should I use the scan method to automatically find the slingboxes (I have 2).  I guess I could enable remote control of my router (but would rather not).

**Comment by GerryDazoo** [comment]
If both your boxes are configured identically then letting slinger scan should work. The original intent was to make it as simple as possible for someone to get started with a single box. Right now I think there would be a race condition where the first box to answer the config broadcast message would be first in the SB list, which might not be correct. But I've been meaning to make multiple boxes work at some point. I'm traveling now and don't have local access to my test boxes but I'll take a look when I get home in a week or two.

**Comment by mdehrlich** [comment]
Thanks.

**Reply by GerryDazoo** [reply]
Give this a try (rename to .py).  Delete the IP address and port number in the config section and replace it with the box's finderid. i.e.
finderid=c631844CF70531865D73650A16A0A536
;ipaddress=192.168.117.114
;port=5201

[sfinder.txt](https://github.com/user-attachments/files/16163158/sfinder.txt)

**Comment by mdehrlich** [comment]
Thanks for working on this.  I am currently traveling and will try it when
I get back.  Might be August.
